### DISCUSSION TITLE
How to automate a topology variation?

### DISCUSSION THREAD

**Discussion by volm92** [InterDigitalInc/AdvantEDGE#255] [Q&A] [answered]
Hello, I would like to carry out a trigger event (e.g. mobility event) without using the graphical interface. I would like to use request.post () in my code for example.
The wiki states that: "AdvantEDGE implements the Sandbox Controller API. This API is used by the GUI and provides the same capabilies; generating event programatically via the REST API is useful in certain scenarios."
However I can't open the link "https://github.com/InterDigitalInc/AdvantEDGE/wiki/api-sandbox-ctrl" and therefore cannot get further details.
Thankyou!

**Comment by dilallkx** [comment, marked as answer]
Hi @volm92,
I just fixed the link in the Wiki.
Thanks!
